src/Request/RequestHandler.php

```diff
@@ -192,6 +193,11 @@ public function request(string $method, string $path, array $parameters = [])
         $options = [
             'headers' => [
                 'Authorization' => $this->getAuthToken(),
+                'User-Agent' => sprintf(
+                  'Clarifai PHP (https://github.com/darrynten/clarifai-php);v%s;%s',
+                  \DarrynTen\Clarifai\Clarifai::VERSION,
```

Comment:
The purpose behind this is so we can better identify how our customers are interacting with our API so we can provide better support. I'm currently trying to get this specific format into all of the awesome libraries from our stellar community.

I can't think of a reason where a user would need to modify this header but one could very well exist. I'd write it off as YAGNI right now but happy to work it in if others need it.